src/main/java/core/basesyntax/model/Account.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.model;
+
+public class Account {
+    private int bananaBalance = 0;
+    private int appleBalance = 0;
+
+    public void setAppleBalance(int appleBalance) {
+        this.appleBalance = appleBalance;
+    }
+
+    public void setBananaBalance(int bananaBalance) {
+        this.bananaBalance = bananaBalance;
+    }
+
+    public int getBananaBalance() {
+        return bananaBalance;
+    }
+
+    public int getAppleBalance() {
+        return appleBalance;
+    }
+
+    public enum Operation {
+        BALANCE("b"),
+        SUPPLY("s"),
+        PURCHASE("p"),
+        RETURN("r");
+        private String code;
+
+        Operation(String code) {
+            this.code = code;
+        }
+
```

Comment:
create static method that throws IllegalArgumentException to parse string into enum (for each loop)

Reply:
I did not understand what I need to fix, can you explain please in another way?

Reply:
fix comment